.github/workflows/desktop

```diff
@@ -0,0 +1,6 @@
+- name: Find issues
```

Comment:
Why the file name is `desktop`? It looks like `find-issues.yml` is more appropriate.

Reply:
have modified it, pl check new commit